gemrb/plugins/SDLVideo/SDLSurfaceSprite2D.cpp

```diff
@@ -103,27 +94,14 @@ void* SDLSurfaceSprite2D::LockSprite()
 	return surface->pixels;
 }
 
-void SDLSurfaceSprite2D::UnlockSprite() const
+void SDLSurfaceSprite2D::UnlockSprite()
 {
 	SDL_UnlockSurface(surface);
-	Invalidate();
-}
-
-bool SDLSurfaceSprite2D::IsPaletteStale() const noexcept
-{
-	// a 'version' implies tha palette was color modified
-	// however, the only caller her is RenderWithFlags
-	// where we already compare the version with the requested one
-	// the version includes the color modification so for now we dont worry about it
-	
-	const Holder<Palette>& pal = format.palette;
-	assert(pal);
-	return pal->GetVersion() != palVersion;
+	invalidate();
```

Comment:
Why lowercase?

Reply:
Wasn't there some kind of private-vs-public kind of convention in some classes? Anyway, I can change it.

Reply:
In some few, mostly in imported code, but definitely not for most. I think it would be better to unify rather than making it more diverse. The one valid exception are iterator accessors, for obvious reasons.